frontend/src/styles/global.scss

```diff
@@ -15,7 +15,10 @@ Only 400 (`normal`), 500 (`var(--font-medium)`), 600 (`var(--font-semibold)`), o
 @import 'functions';
 @import 'mixins';
 @import 'utility-legacy';
-@import 'utilities';
+@import 'colors';
+@tailwind base;
```

Comment:
I believe this is overriding some global styles that we have previously relied on from Antd. 

In particular standard headings have some styles applied that seem to be reset in the tailwind base.
Generally I think this is a good thing, as the inherited antd styles are pretty obscure, we just need to add our own here.

<img width="460" alt="Screenshot 2023-12-06 at 09 31 33" src="https://github.com/PostHog/posthog/assets/2536520/820c1130-589a-4dca-89be-f42c4b848d9b">

Reply:
I like Tailwind's philosophy with the normalization… but we can move to it separately from the utility classes